politeiad/backendv2/tstorebe/plugins/pi/hooks.go

```diff
@@ -249,6 +294,25 @@ func (p *piPlugin) proposalFilesVerify(files []backend.File) error {
 		}
 	}
 
+	// Validate vote & proposal metadata requirements
+	vm, err := voteMetadataDecode(files)
+	if err != nil {
+		return err
+	}
+	// In case of a RFP ensure irrelevant proposal metadata are not provided.
+	isRFP := vm != nil && vm.LinkBy != 0
```

Comment:
Can we put this into a `isRFP` function instead.  This check is something that is likely to be re-used so its better to have a single function instead of checking these fields manually every time.  The definition of what makes something an RFP could potentially change in the future as well so having a function allows you to make the update in a single location instead of having to find all the instances in the code of this check.

Reply:
👍🏽